File: 151004/Bosko/lab2/lab2/DTO/UserResponseTo.cs
using lab2.DTO.Interface;
using lab2.Models;

namespace lab2.DTO
{
    public class UserResponseTo(int Id, string? Login, string? Password, string? Firstname, string? Lastname, ICollection<News> News) : IResponseTo
    {
        public int Id { get; set; } = Id;

        public string? Login { get; set; } = Login;

        public string? Password { get; set; } = Password;

        public string? Firstname { get; set; } = Firstname;

        public string? Lastname { get; set; } = Lastname;

        public ICollection<News> News { get; set; } = News;
    }
}

File: 151004/Gonzarevich/lab1/DTO/UserResponseTo.php
<?php

namespace App\DTO;

use App\Entity\User;

class UserResponseTo
{
    public readonly int $id;
    public readonly string $login;
    public readonly string $password;
    public readonly string $firstname;
    public readonly string $lastname;

    public function __construct(
        User $user
    ) {
        $this->id = $user->getId();
        $this->login = $user->getLogin();
        $this->firstname = $user->getFirstname();
        $this->lastname = $user->getLastname();
    }
}